luigi/local_target.py

```diff
@@ -71,7 +71,12 @@ def mkdir(self, path, parents=True, raise_if_exists=False):
                 return
 
         if parents:
-            os.makedirs(path)
+            try:
+                os.makedirs(path)
+            except OSError as err:
+                # somebody already created the path
+                if err.errno != errno.EEXIST:
```

Comment:
I think it's correct. https://docs.python.org/3.6/library/errno.html#errno.EEXIST